src/gui/tray/UserModel.cpp

```diff
@@ -413,15 +413,29 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+bool User::isValueableActivity(const Folder *folder, const SyncFileItemPtr &item) const
+{
+    // Ignore activity from a different account
+    if (folder->accountState() != _account.data()) {
+        return false;
+    }
+
+    // We just care about conflict issues that we are able to resolve
+    if (item->_status == SyncFileItem::Conflict && !Utility::isConflictFile(item->_file)) {
+        return false;
+    }
+
+    return true;
+}
+
 void User::slotItemCompleted(const QString &folder, const SyncFileItemPtr &item)
 {
     auto folderInstance = FolderMan::instance()->folder(folder);
 
     if (!folderInstance)
         return;
 
-    // check if we are adding it to the right account and if it is useful information (protocol errors)
-    if (folderInstance->accountState() == _account.data()) {
+    if (isValueableActivity(folderInstance, item)) {
```

Comment:
Ouch, this is subpar API naming, what is "valueable"? It will tell me nothing months from now. I think it was better not factoring this out in a method but instead store the result in two nicely named booleans and combine then in the if. Typically something like:
```cpp
const auto isForUserAccount = folderInstance->accountState() == _account.data();
const auto isUnresolvableConflict = item->_status == SyncFileItem::Conflict && !Utility::isConflictFile(item->_file);
if (isForUserAccount && !isUnresolvableConflict) {
```

Reply:
@FlexW I guess we still have time to create a PR with just these changes :) @er-vin Sorry, we had to make it move forward. I must confess, I am not that nitpicky reviewer :)

Reply:
@er-vin @allexzander pr for that https://github.com/nextcloud/desktop/pull/2976

Reply:
> @er-vin Sorry, we had to make it move forward.

No worries, I'm less available that's how it is. :-)

> I must confess, I am not that nitpicky reviewer :)

Well, I know I put quite some nitpicks too, but there it was really more of a code smell. If something is not clearly named for some reason and one can't find a proper name for it... well something is fishy. ;-)